src/data/iconData.json

```diff
@@ -0,0 +1,32 @@
+{
+  "summary": {
+    "200": "https://cdn-icons-png.flaticon.com/128/365/365857.png",
+    "300": "https://cdn-icons-png.flaticon.com/128/365/365860.png",
+    "500": "https://cdn-icons-png.flaticon.com/128/365/365859.png",
+    "600": "https://cdn-icons-png.flaticon.com/128/365/365925.png",
+    "700": "https://cdn-icons-png.flaticon.com/128/365/365867.png",
+    "800": "https://cdn-icons-png.flaticon.com/128/365/365851.png",
+    "801": "https://cdn-icons-png.flaticon.com/128/365/365916.png",
+    "802": "https://cdn-icons-png.flaticon.com/128/365/365916.png",
+    "803": "https://cdn-icons-png.flaticon.com/128/365/365916.png",
+    "804": "https://cdn-icons-png.flaticon.com/128/365/365916.png"
```

Comment:
Since these are all the same, theres no need for them all to be here. You could delete most of them and handle the choice in component instead